### PR TITLE
release v1.5.6

### PR DESCRIPTION
**Pull request by hlts2**
As titled

### CODE REVIEW

File: content/docs/overview/component/filter-gateway.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Filter Gateway_overview/Component"
-date: 2022-07-19T14:38:15+09:00
-draft: false
-weight: 300
+date: 2022-07-21T10:34:24+09:00
+draft: true
+weight: 0
```
Comment (vankichi) [inline content/docs/overview/component/filter-gateway.md:5]:
```suggestion
draft: false
weight: 300
```

File: content/docs/v1.5/overview/component/filter-gateway.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Filter Gateway_v1.5/Overview/Component"
-date: 2022-07-19T14:38:23+09:00
-draft: false
-weight: 300
+date: 2022-07-21T10:34:26+09:00
+draft: true
+weight: 0
```
Comment (vankichi) [inline content/docs/v1.5/overview/component/filter-gateway.md:5]:
```suggestion
draft: false
weight: 300
```

File: content/docs/v1.5/release/changelog.md [hunk 3]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Changelog_v1.5/Release"
-date: 2022-07-05T03:01:22Z
-draft: false
+date: 2022-07-21T10:34:26+09:00
+draft: true
```
Comment (vankichi) [inline content/docs/v1.5/release/changelog.md:4]:
```suggestion
draft: false
```

File: content/docs/release/changelog.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 title: "Changelog_release"
-date: 2022-07-05T03:01:15Z
-draft: false
+date: 2022-07-21T10:34:25+09:00
+draft: true
```
Comment (vankichi) [inline content/docs/release/changelog.md:4]:
```suggestion
draft: false
```

**Review by vankichi — CHANGES_REQUESTED** [review]

File: content/docs/overview/component/filter-gateway.md [hunk 5]
```diff
@@ -26,6 +26,7 @@ Ingress filtering is the pre-process before the request is processed by the Vald
 This is useful, e.g., when you want to vectorize object data (when you want to handle object data directly as the request query) or when you want to pad the vector, etc.
 
 Here are the steps of pre-processing with the ingress filter:
+
```
Comment (kevindiu) [inline content/docs/overview/component/filter-gateway.md:29]: seems the image below is dead.
https://github.com/vdaas/web/pull/116/files#diff-9f4fffa70fce0b833f4ccbade8f2f5bcc5c2715d57e672a8e4af6dfa768f031aR34-R36

Comment (vankichi) [reply content/docs/overview/component/filter-gateway.md:29]: ???

Comment (kevindiu) [reply content/docs/overview/component/filter-gateway.md:29]: sorry my mistake

**Comment by hlts2** [comment]
📝 I checked the web page using a [stating environment](https://vald.netlify.app/) and there were no problems.

**Review by vankichi — APPROVED** [review]
LGTM. Thanks :pray:

**Review by kpango — APPROVED** [review]

**Comment by hlts2** [comment]
@vankichi @kpango 
Thank you for your review!
I will merge this PR 👍

**Pull request merged**